airbyte-integrations/connectors/destination-s3-data-lake/src/test-integration/kotlin/io/airbyte/integrations/destination/s3_data_lake/S3DataLakePerformanceTest.kt

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.s3_data_lake
+
+import io.airbyte.cdk.load.command.aws.asMicronautProperties
+import io.airbyte.cdk.load.write.BasicPerformanceTest
+import java.nio.file.Files
+
+class S3DataLakePerformanceTest :
+    BasicPerformanceTest(
+        configContents = Files.readString(S3DataLakeTestUtil.GLUE_CONFIG_PATH),
+        configSpecClass = S3DataLakeSpecification::class.java,
+        defaultRecordsToInsert = 500_000,
+        micronautProperties =
+            S3DataLakeTestUtil.getAwsAssumeRoleCredentials().asMicronautProperties(),
```

Comment:
I'm still confused about why this is needed - e.g. if I remove this line from the tests locally + run GlueWriteTest, that works fine (and obviously GlueAssumeRoleWriteTest fails)